src/shared/utils/app/show-avatars.ts

```diff
@@ -0,0 +1,7 @@
+import { UserService } from "../../services";
+
+export default function showAvatars(
+  myUserInfo = UserService.Instance.myUserInfo
+): boolean {
```

Comment:
I'm curious about these functions that take myUserInfo as an arg. Do they ever use anything other than the myUserInfo from the UserService that's set as default?

Reply:
> I'm curious about these functions that take myUserInfo as an arg. Do they ever use anything other than the myUserInfo from the UserService that's set as default?

Good point. I'd love to go through and clean up all these functions in the next PR. It's be a lot easier to clean them out now that they're all separate files.